docs/sources/query/logcli.md

```diff
@@ -400,6 +413,76 @@ Args:
   <query>  eg '{foo="bar",baz=~".*blip"} |~ ".*error.*"'
 ```
 
+### LogCLI instant-query command reference
```

Comment:
nit: Could we mark the query type in all the headings as code and remove the redundant "LogCLI" prefix, as this would shorten the ToC on the right side of the page.

```suggestion
### `instant-query` command reference
```

Reply:
Done !